pkg/testutil/leak.go

```diff
@@ -113,6 +113,7 @@ func interestingGoroutines() (gs []string) {
 			strings.Contains(stack, "github.com/coreos/etcd/pkg/testutil.interestingGoroutines") ||
 			strings.Contains(stack, "github.com/coreos/etcd/pkg/logutil.(*MergeLogger).outputLoop") ||
 			strings.Contains(stack, "github.com/golang/glog.(*loggingT).flushDaemon") ||
+			strings.Contains(stack, "github.com/coreos/etcd/integration.TestV3ClientMetrics") ||
```

Comment:
please don't mask goroutine leaks for specific test cases; leak checking is incredibly useful for debugging and there's usually a way to write the test so it won't leak